plugin/storage/cassandra/spanstore/dbmodel/tag_filter_exact_match_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2019 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dbmodel
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/jaegertracing/jaeger/model"
+)
+
+func TestBlacklistFilter(t *testing.T) {
+	// expected
+	tt := [][][]string{
```

Comment:
nit: make this more readable by using struct:
```
tests := []struct{
    input []string
    filter []string
    expected []string
}{
  {
    input: ...
    filter: ...
  },
}
```

Reply:
Agreed and I appreciate the patience.  I'm relatively new to go.